metaflow/plugins/kfp/tests/setup.cfg

```diff
@@ -0,0 +1,16 @@
+[coverage:run]
+omit = ./*
```

Comment:
how does this get coverage for what is under `/metaflow/metaflow/plugins/kfp`?

Reply:
@talebzeghmi , it works as follows:
If you look at the changes in the `.gitlab-ci.yml` file, you'll see `--cov=../ --cov-config=setup.cfg`. The `--cov=../` indicates we'll provide coverage for the directory one level above from where the `.gitlab-ci.yml` file is located (`/metaflow/metaflow/plugins/kfp/tests` (`.gitlab-ci.yml` file is located) -> `/metaflow/metaflow/plugins/kfp`).
The `omit = ./*` indicates we will not provide coverage for any files within the same directory as where `setup.cfg` is located (`/metaflow/metaflow/plugins/kfp/tests`).

This prevents us from providing coverage from sample flows and integration testing code and focus only on kfp plugin code. Is that clear?

Reply:
Can the `--cov=../` be provided in setup.cfg next to the `omit=` statement?  That'd make it easier to find and follow